### DISCUSSION TITLE
What’s the right way count the amount of characters in editor and prevent user from typing if above a threshold?

### DISCUSSION THREAD

**Discussion by trancethehuman** [yoopta-editor/Yoopta-Editor#337] [Q&A]
I’m thinking of just checking on change but not sure if there’s a better way. Also not sure how to prevent user from typing more without disabling the whole editor (perhaps reject keystrokes?)

**Comment by Darginec05** [comment]
hmm

**Comment by Darginec05** [comment]
Could you please create issue with [Example Request] template?
I'll prepare example for this case
